### ISSUE TITLE
mac上ollama部署2.5版本，聊天一直报错

### ISSUE THREAD

**Issue by zhb-code** [OpenBMB/ollama#18]
### What is the issue?

mac上ollama部署2.5版本，聊天一直报错，错误信息：llama_get_logits_ith: invalid logits id 12, reason: no logits
![1723028561872](https://github.com/user-attachments/assets/29a68d6a-1d82-434f-8930-428cab773937)

### OS

macOS

### GPU

Apple

### CPU

Apple

**Comment by hooploop** [comment]
能问一下如何解决的么？

**Comment by zhb-code** [comment]
没有解决，我删掉之前配置好的ollama，然后重新拉取的一份，按照文档步骤执行，编译都过不了了，现在卡另一个缺库问题了，哎！

---- 回复的原邮件 ----
| 发件人 | John ***@***.***> |
| 发送日期 | 2024年08月09日 14:32 |
| 收件人 | OpenBMB/ollama ***@***.***> |
| 抄送人 | zhb-code ***@***.***>,
State ***@***.***> |
| 主题 | Re: [OpenBMB/ollama] mac上ollama部署2.5版本，聊天一直报错 (Issue #18) |

能问一下如何解决的么？

**Comment by zhb-code** [comment]
我关掉这个是官方让去主问题区域提问
